tools/nl/embeddings/build_embeddings.py

```diff
@@ -269,17 +229,8 @@ def main(_):
 
   if use_finetuned_model:
     ctx_no_model = utils.Context(gs=gs, model=None, bucket=bucket, tmp='/tmp')
-
-    # Check if this model is already downloaded locally.
-    if os.path.exists(os.path.join(ctx_no_model.tmp, model_version)):
-      tuned_model_path = os.path.join(ctx_no_model.tmp, model_version)
-      print(f"Model already downloaded at path: {tuned_model_path}")
-    else:
-      print("Model not previously downloaded locally. Downloading from GCS.")
-      tuned_model_path = utils.download_model_from_gcs(ctx_no_model,
-                                                       model_version)
-      print(f"Model downloaded locally to: {tuned_model_path}")
-
+    tuned_model_path = utils.get_or_download_model_from_gcs(
+        ctx_no_model, model_version)
     model = SentenceTransformer(tuned_model_path)
```

Comment:
nit: either call get_ft_model_from_gcs here, or drop the helper and just call SentenceTransformer() in build_custom_embeddings.py (since a wrapper func just has one call site and one additional line)

Reply:
Makes sense. Calling `get_ft_model_from_gcs` from here.